README.md

```diff
@@ -1852,6 +1852,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Rana Muskan](https://github.com/theranamuskan)
   - [Randhir Shaw](https://github.com/randhirshaw)
   - [Rashmi Wijesekara](https://github.com/Rashmi-Wijesekara)
+  - [Rashini Gamalath](https://github.com/Rashini1998)
```

Comment:
```suggestion
  - [Rashini Gamalath](https://github.com/Rashini1998)
  - [Rashmi Wijesekara](https://github.com/Rashmi-Wijesekara)
```

Reply:
@Rashini1998 Kindly arrange your full name chronologically in order to ensure uniformity and simplicity of access.